dev-packages/application-manager/src/generator/backend-generator.ts

```diff
@@ -78,9 +78,9 @@ BackendApplicationConfigProvider.set(${this.prettyStringify(this.pck.props.backe
 
 const serverModule = require('./server');
 const serverAddress = main.start(serverModule());
-serverAddress.then(function ({ port, address }) {
+serverAddress.then(function ({ port }) {
     if (process && process.send) {
-        process.send({ port, address });
+        process.send(port);
```

Comment:
How this change is related? We can break some clients which expect to get address like tests: https://github.com/eclipse-theia/theia/blob/e876b16eb8600a944d69d8ed742c79b90924b7f5/dev-packages/cli/src/run-test.ts#L65

Reply:
Yes.